_config.yml

```diff
@@ -34,6 +34,21 @@ collections:
   blog:
     output: "true"
     permalink: /blog/:year/:month/:slug
+  events:
+    output: "true"
+    permalink: /events/:year/:month/:title
+
+defaults:
+  - scope:
+      path: ""
```

Comment:
This looks like it would be possible to use different folders for blog posts and events. If that's the case, I think that would be nicer :)